mucgpt-frontend/src/components/ChatLayout/ChatLayout.module.css

```
@@ -154,14 +162,45 @@
     min-height: 0;
     overflow-y: auto;
     -webkit-overflow-scrolling: touch;
     box-sizing: border-box;
-    padding: 8px 24px;
+    margin: 0;
+    padding: 0 24px;
+    list-style: none;
+    mask-image: linear-gradient(to bottom, transparent 0, #000 var(--chatFadeTop), #000 calc(100% - var(--chatFadeBottom)), transparent 100%);
+    -webkit-mask-image: linear-gradient(to bottom, transparent 0, #000 var(--chatFadeTop), #000 calc(100% - var(--chatFadeBottom)), transparent 100%);
+    transition:
+        --chatFadeTop 160ms ease,
```

Stylelint (17.14.1)

[error] Expected "--chatFadeTop" to be "--chatfadetop" (value-keyword-case)

(value-keyword-case)